src/main/java/com/baidu/hugegraph/util/CollectionUtil.java

```diff
@@ -84,4 +84,82 @@ public static boolean allUnique(Collection<?> collection){
         }
         return new LinkedHashSet<>(list.subList(from, to));
     }
+
+    public static <T> Set<T> union(Collection<T> first, Collection<T> second) {
+        E.checkNotNull(first, "first");
+        E.checkNotNull(second, "second");
+        HashSet<T> results = new HashSet<>(first);
+        results.addAll(second);
+        return results;
+    }
+
+    /**
+     * Find the intersection of two collections, the original collections
+     * will not be modified
+     * @param first the first collection
+     * @param second the second collection
+     * @param <T> element type of collection
+     * @return intersection of the two collections
+     */
+    @SuppressWarnings("unchecked")
```

Comment:
prefer to move to line 112

Reply:
Move to line 112 will tip "Annotations are not allowed here"